.github/workflows/compile_changelogs.yml

```diff
@@ -2,7 +2,7 @@ name: Compile changelogs
 
 on:
   schedule:
-  - cron: "5 5 * * *"
+  - cron: "0 0 * * *"
```

Comment:
You shouldn't have done this.

![image](https://user-images.githubusercontent.com/35135081/116190629-d700be00-a6df-11eb-8380-03ce661e6ca6.png)

Literally all you had to do was add [`workflow_dispatch`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch), and we could've run it manually.